package.json

```diff
@@ -85,7 +85,6 @@
     "@storybook/react-native": "^4.1.1",
     "babel-jest": "^23.6.0",
     "babel-plugin-jest-hoist": "^23.2.0",
-    "bpk-svgs": "^6.1.0",
```

Comment:
I removed the top level version as it seems superfluous ... any ideas?

Reply:
Seems reasonable. If any of our packages are relying on it they should have it themselves